src/eko/evolution_operator/__init__.py

```diff
@@ -242,16 +243,16 @@ def labels(self):
             logger.warning("Evolution: skipping non-singlet sector")
         else:
             # add + as default
-            labels.append("NS_p")
+            labels.append((br.non_singlet_pids_map["ns+"], 0))
```

Comment:
All these tuples `(br.non_singlet_pids_map["ns+"], 0)` are already in `non_singlet_labels `. Can't you use the second one directly?

Reply:
Yes of course. Anyway I think that the first one is a bit more readable than the second.

Reply:
you are right, I just found it quite long, since it appears many times... in the end to singlet you're using `singlet_labels` straight